plugins/contentful.js

```diff
@@ -2,13 +2,17 @@ const contentful = require('contentful');
 
 // These values will be set via env vars.
 // If this file is imported these values may be available to the client.
-const config = {
+let config = {
```

Comment:
I think we can still use a CONST here as we are not re-assigning or re-declaring the variable, however, I'm sure your eslint is probably more accurate